rbs/rbs_common.cc

```diff
@@ -2,11 +2,50 @@
 
 namespace sorbet::rbs {
 
-core::LocOffsets locFromRange(core::LocOffsets loc, const rbs_range_t &range) {
+core::LocOffsets RBSDeclaration::commentLoc() const {
     return {
-        loc.beginPos() + range.start.char_pos,
-        loc.beginPos() + range.end.char_pos,
+        comments.front().commentLoc.beginLoc,
+        comments.back().commentLoc.endLoc,
     };
 }
 
+std::string RBSDeclaration::string() const {
+    std::string result;
+    for (const auto &comment : comments) {
+        result += comment.string;
+    }
+    return result;
+}
```

Comment:
I don't know whether you want to compute these eagerly in the constructor. If it's the case that every time an `RBSDeclaration` is created we're going to need the `string()`, it would be costly to allocate the joined strings twice.

The loc ones are ~fine`, because those are quick to compute and don't really allocate memory.

Reply:
I think the original idea was to initialize the declaration then to append directly to the comments vector but in practice we never append. Changed in https://github.com/sorbet/sorbet/pull/8760.